### DISCUSSION TITLE
Jupiter notebook  Markdown Korean Broken problem

### DISCUSSION THREAD

**Discussion by ghost** [community/community#37294] [Other Feature Feedback, Questions, & Ideas]
Hello, I'm pushing the code of Jupiter's laptop, but the Korean markdown keeps breaking. Not only my code but also other user codes seem to be broken, where is the problem?

**Comment by ossanna16** [comment]
Hi @NohJeongKim,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Jupyter Notebook. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
